app.js

```diff
@@ -1,24 +1,34 @@
-const express = require("express")
-const {getTopics, getApi, getArticlesById, getAllArticles} = require("./controllers/controller.js")
-const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./errors/errors.js")
+const express = require("express");
+const {
+  getTopics,
+  getApi,
+  getArticlesById,
+  getAllArticles, getCommentByArticleId
+} = require("./controllers/controller.js");
+const {
+  handlePsqlErrors,
+  handleCustomErrors,
+  handleServerErrors,
+} = require("./errors/errors.js");
 
+const app = express();
+app.use(express.json());
```

Comment:
We have no need for this line of code yet as we aren't currently handling any post requests - please remove.